tests/test_liquidations.py

```diff
@@ -0,0 +1,35 @@
+# from typing import List
+
+# from mev_inspect.compound_liquidations import get_cether_liquidations
+# from mev_inspect.schemas.liquidations import Liquidation
+# from mev_inspect.schemas.classified_traces import Protocol
+# from mev_inspect.classifiers.trace import TraceClassifier
+# from tests.utils import load_test_block
+
+
+# def test_single_weth_liquidation():
+
+#     # transaction_hash = (
+#     #     "0xc6a56faeba98a0ac37a83f57d460f46d8acc4ebdf683e422f6e606f7cdb0fa59"
+#     # )
+#     # block_number = 12763965
+
+#     # liquidations = [
+#     #     Liquidation(
+#     #         liquidated_user="0xcf35b698fa1a412de754dd807d89e6c207eb55fb",
+#     #         liquidator_user="0x681bd23f6128db3f9b8914595d1a63830a6212fa",
+#     #         collateral_token_address="0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+#     #         debt_token_address="0x6c8c6b02e7b2be14d4fa6022dfd6d75921d90e4e",
+#     #         debt_purchase_amount=17921034103282692,
+#     #         received_amount=359568053354,
+#     #         protocol=Protocol.compound,
+#     #         transaction_hash=transaction_hash,
+#     #         block_number=block_number,
+#     #     )
+#     # ]
+
+#     # block = load_test_block(block_number)
+#     # trace_classifier = TraceClassifier()
+#     # classified_traces = trace_classifier.classify(block.traces)
+#     # result = get_cether_liquidations(classified_traces)
+#     # assert(result == liquidations)
```

Comment:
what's going on here?

Reply:
had to comment out until I got cache files for a few blocks of out the kube setup (as they'd fail), fixed that, will push new commits + rebase off main to use the common liquidation types